tests/Authzed.Net.Tests/V1Test.cs

```diff
@@ -1,12 +1,334 @@
 namespace Authzed.Net.Tests;
 
 using Authzed.Api.V1;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
 
 public class V1Test
 {
     [Fact]
-    public void TestBasicSchema()
+    public async Task TestBasicSchema()
```

Comment:
I made all of the tests async under the assumption that it'll make them generally faster to run. Currently the whole test suite runs in like 250ms, so it may not actually make that much of a difference.